docs/installation/docker.md

```diff
@@ -0,0 +1,109 @@
+
+
+## Launching Islandora with Docker
+
+In a shell, clone the isle-dc project. In that directory, enter the command `make` (or `make dev` - see note) to build the docker infrastructure. Then use the command `docker-compose up -d` to start the containers.
+
+!!! Note
+    The `make` command alone will spin up a sandbox-like version of Isle on the front end, but the code files will be inaccessible. The `make dev` command will copy the files locally in a way that is live to the Isle site. This method takes longer, but is required if you will be testing pull requests or writing code. 
+
+```bash
+clone https://github.com/islandora-devops/isle-dc
+cd isle-dc
+make
+```
+
+
+Results:
+```
+isle-dc$ make
+  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                 Dload  Upload   Total   Spent    Left  Speed
+100  3562  100  3562    0     0  12779      0 --:--:-- --:--:-- --:--:-- 12812
+  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                 Dload  Upload   Total   Spent    Left  Speed
+100  1708  100  1708    0     0   6809      0 --:--:-- --:--:-- --:--:--  6832
+docker-compose pull
+Pulling activemq   ... done
+Pulling alpaca     ... done
+Pulling blazegraph ... done
+Pulling cantaloupe ... done
+Pulling fcrepo     ... done
+Pulling fits       ... done
+Pulling crayfits   ... done
+Pulling gemini     ... done
+Pulling homarus    ... done
+Pulling houdini    ... done
+Pulling hypercube  ... done
+Pulling mariadb    ... done
+Pulling matomo     ... done
+Pulling milliner   ... done
+Pulling recast     ... done
+Pulling solr       ... done
+Pulling drupal     ... done
+Pulling traefik    ... done
+Pulling watchtower ... done
+```
+
+!!! Troubleshooting
+    If you get an error such as: `ERROR: Version in "./docker-compose.activemq.yml" is unsupported.`, then you need to upgrade Docker, and then enter the command `make clean`. If you forget to `make clean`, then the next time `make` runs you may see an error such as: `ERROR: Top level object in './docker-compose.yml' needs to be an object not '<class 'NoneType'>'.`
+
+!!! Troubleshooting
+    If you get an error such as: `make[1]: *** [create-codebase-from-demo] Error 143 make: *** [dev] Error 21` ... documentation to come.
+
+
+Once `make` has successfully completed, launch the Isle containers using `docker-compose up`. The `-d` flag allows you to return to using the command line. Without it, your shell will be stuck in the `docker-compose` process as long as the containers are running.
+
+
+```bash
+docker-compose up -d
+```
+
+## Visiting your Islandora site
+
+Direct a browser to [https://islandora-isle-dc.traefik.me/](https://islandora-isle-dc.traefik.me/) (yes, it's awkward and we hope to resolve that). Your first time, you will probably see a white screen. This will persist until all containers are successfully warmed up. Then, you should see a basic Drupal login screen.
```

Comment:
Maybe add expected wait time? I seem to recall @dannylamb said 2-3 minutes for the white screen to resolve.

Reply:
it seemed to take longer than that - but yes! I'd also like to add the command to open the log that showed whether "stuff was happening", but (1) I can't remember which log that was - `docker logs` or `docker-compose logs` - and what the "tell" was that it wasn't finished working.